pysnark/boolean.py

```diff
@@ -150,6 +178,11 @@ def __bool__(self):
             Instead of if statements, use if_then_else from pysnark.branching")
         
     def __pow__(self, other, mod=None):
+        """
+        Raises a LinCombBool to the power of an integer, LinComb, or LinCombBool
+        Returns a LinComb
+        Costs 36 constraints
+        """

```

Comment:
For boolean values b (being 0 or 1), if b=0 then b**x=0 for any x, and if b=1 then b**x=1 for any x. So I think this can be implemented as (b!=0)